simplified-ui-accounts/src/main/java/org/nypl/simplified/ui/accounts/AccountListAdapter.kt

```diff
@@ -29,9 +28,9 @@ class AccountListAdapter(
     viewType: Int
   ): AccountViewHolder {
     val inflater = LayoutInflater.from(parent.context)
-    val itemView = inflater.inflate(R.layout.account_list_item_old, parent, false)
+    val binding = AccountListItemBinding.inflate(inflater, parent, false)
```

Comment:
I've always seen some end of life management with view bindings. Isn't it necessary?

Reply:
I believe that should only be necessary when using LiveData in your bindings, so not needed here.